src/main/java/com/discordbolt/api/command/CommandContext.java

```diff
@@ -18,13 +19,21 @@
     private IMessage message;
     private List<String> arguments;
     private CustomCommand customCommand;
+    private String prefix;
+    private IDiscordClient client;
 
-    protected CommandContext(IMessage message, CustomCommand customCommand) {
+    protected CommandContext(IMessage message, CustomCommand customCommand, IDiscordClient client, String prefix) {
```

Comment:
`IDiscordClient` can be fetched from the `IMessage`, so the `CommandContext#getClient()` method should be `return message.getClient()`

Reply:
Passing in the String prefix is redundant, the method to get the prefix should query `CommandManager#getCommandPrefix(IGuild)`